### ISSUE TITLE
Error getting value in HW706

### ISSUE THREAD

**Issue by Bekosantux** [jlennox/HeartRate#50]
Hi.
Thank you for a great app. I have been looking for this.
However, I encountered a problem. At first it worked perfectly, but when I switch off the band, the app detects the disconnection and then switch on the band again, I get 'Attempt to configure service failed'.
The logs show that the device is recognised but fails to retrieve values, so I have to repeat the pairing several times via Bluetooth to get it working again, but this is very unstable and time consuming.

[logs.txt](https://github.com/jlennox/HeartRate/files/14784422/logs.txt)

**Comment by jlennox** [comment]
This is interesting.

The [WriteClientCharacteristicConfigurationDescriptorAsync](https://learn.microsoft.com/en-us/uwp/api/windows.devices.bluetooth.genericattributeprofile.gattcharacteristic.writeclientcharacteristicconfigurationdescriptorasync) API keeps returning Unreachable, which means [No communication can be performed with the device, at this time.](https://learn.microsoft.com/en-us/uwp/api/windows.devices.bluetooth.genericattributeprofile.gattcommunicationstatus?view=winrt-22621)

If you switch off the band, close this program. Switch the band back on, then reopen this program, does it work?

To be honest, my knowledge of bluetooth LE was never much, and what I did have has atrophied over the years. If you have any programming experience, your best bet might be to debug this yourself,but I can help you the best I can.

**Comment by Bekosantux** [comment]
Thank you for your kind response. We are pleased to inform you that the problem has been resolved.
The problem was in Windows, google showed a lot of people experiencing the same symptoms with game controllers etc. I tried many things, including installing drivers, but the direct solution was to check 'Bluetooth -> More bluetooth options -> Allow Bluetooth devices to find this PC'.
Now I can use the app stress-free. Thank you.
